### PR TITLE
chore(core): Remove all references to AnchorProof.blockTimestamp (or blockNumber)

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

File: packages/core/src/conflict-resolution.ts [hunk 1]
```diff
@@ -99,10 +98,14 @@ export async function pickLogToAccept(
       )
     }
 
-    // Compare block heights to decide which to take
-    if (proof1.blockNumber < proof2.blockNumber) {
+    // Compare anchor block timestamp to decide which to take.  Even though blockTimestamps can
+    // drift from wallclock time, they are guaranteed to be increasing in increasing blockNumbers,
+    // so they are still safe to compare for relative ordering.
```
Comment (oed) [inline packages/core/src/conflict-resolution.ts:103]: Is this enforced by ethereums consensus logic? Maybe add a link to some reference here?

**Review by oed — APPROVED** [review]
Should we do anything to the StreamState AnchorProof ? Do we need this anymore?

**Comment by oed** [comment]
I see.

**Comment by stbrody** [comment]
closing in favor of https://github.com/ceramicnetwork/js-ceramic/pull/2623

**Pull request closed without merging**